fuzz.go

```diff
@@ -33,7 +33,10 @@ func Fuzz(data []byte) int {
 		if err != nil {
 			// must succeed, as obj was obtained via successful decode
 			// the only exception is that we cannot encode non-string Ref at proto=0
```

Comment:
This comment is now out of date

Reply:
True, thanks for catching. I will post corrected patch.